webpack/config.json

```diff
@@ -3,6 +3,6 @@
     "apiHost": "http://localhost:3004"
   },
   "production": {
-    "apiHost": ""
+    "apiHost": "/deposit-api"
```

Comment:
@lindareijnhoudt : apiHost = /deposit-api ? Is this correct?

Reply:
yes, this works. Since this starts with `/` rather than some base url, it will take the base url from it's own host.